Python/Product/Analysis/Interpreter/Ast/AstPythonInterpreterFactory.cs

```diff
@@ -194,6 +194,10 @@ bool useDefaultDatabase
 
             var hash = SHA256.Create();
             var dir = Path.GetDirectoryName(filePath);
+            if (IsWindows()) {
```

Comment:
Whatever problem you are fixing will still exist on Linux/Mac?

Reply:
Paths are case sensitive on Linux/Mac, so we won't resolve the imports if the casing doesn't match.